stats/lib/platform-stats-fetchers.js

```diff
@@ -53,3 +57,17 @@ export const fetchDailyRewardTransfers = async (pgPool, filter) => {
   `, [filter.from, filter.to])
   return rows
 }
+
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchDailyWalletCount = async (pgPool, filter) => {
+  return await getDailyDistinctCount({
+    pgPool,
+    table: 'daily_reward_transfers',
+    column: 'to_address',
+    filter,
+    asColumn: 'wallet_count'
```

Comment:
Since this reports the number of wallets that received rewards payout on a given day, I feel the names "daily wallet count" and "wallet_count" are a bit misleading.

I think more accurate names would be something like "daily recipients of rewards" and "recipient_count".

Thoughts?